server/channels/store/sqlstore/channel_store.go

```diff
@@ -3071,7 +3072,7 @@ func (s SqlChannelStore) Autocomplete(userID, term string, includeDeleted, isGue
 	}
 
 	channels := model.ChannelListWithTeamData{}
-	err = s.GetReplicaX().Select(&channels, sql, args...)
+	err = s.GetSearchReplicaX().Select(&channels, sql, args...)
```

Comment:
To confirm, the search replica in a non-ES deployment is "expected" to be hammered by post search, with the theory being that if that gets offloaded (but is still slow), the rest of the application will feel more responsive.

But this particular channel search operation powers the quick switcher, which I imagine gets used at least an order of magnitude more often. Are we confident we should offload this operation to the search replicas as well?

Reply:
Okay that's a good point. I'll revert this change.